Civilla-Project/Police-Control/src/main/java/CivillaDatabaseDummyConnector.java

```diff
@@ -0,0 +1,38 @@
+import sun.reflect.generics.reflectiveObjects.NotImplementedException;
+
+import java.util.ArrayList;
+import java.util.Hashtable;
+
+public class CivillaDatabaseDummyConnector implements ICivillaDatabase {
+
+    private static final Hashtable<String, CivillaDatabaseItem> databaseItems =
+            new Hashtable<>();
+
+    @Override
+    public CivillaDatabaseResponse get(String id) {
+        if (databaseItems.getOrDefault(id, null) != null){
+            ArrayList<CivillaDatabaseItem> ret = new ArrayList<>();
+            ret.add(databaseItems.get(id));
+            return new CivillaDatabaseResponse(200, "ok", ret);
+        }
+        return new CivillaDatabaseResponse(404, "not found", new ArrayList<>());
+    }
+
+    @Override
+    public CivillaDatabaseResponse put(CivillaDatabaseItem item) {
+        databaseItems.put(item.id, item);
+        return new CivillaDatabaseResponse(200, "ok");
+    }
+
+    @Override
+    public CivillaDatabaseResponse delete(String id) {
+        throw new NotImplementedException();
+    }
+
+    @Override
+    public CivillaDatabaseResponse query(String filter) {
+        if (databaseItems.isEmpty())
+            return new CivillaDatabaseResponse(404, "not_found", new ArrayList<CivillaDatabaseItem>(databaseItems.values()));
+        return new CivillaDatabaseResponse(200, "ok", new ArrayList<CivillaDatabaseItem>(databaseItems.values()));
```

Comment:
Is this implementation completed? If not it is better use NotImplementedException.

Reply:
By now it is just dummy connector with local storage. Where exactly do u suggest to put NotImplemented?

Reply:
for dummy query() call its ok